### PR TITLE
Rate changes to stay within the rate limits mentioned in #215

### PR DESCRIPTION
**Pull request by asd20752**
Rate limit changes to keep within the rate limits mentioned in issue #215

### CODE REVIEW

File: custom_components/skodaconnect/const.py [hunk 1]
```diff
@@ -29,8 +29,8 @@
 
 SIGNAL_STATE_UPDATED = f"{DOMAIN}.updated"
 
-MIN_SCAN_INTERVAL = 10
-DEFAULT_SCAN_INTERVAL = 60
+MIN_SCAN_INTERVAL = 80
```
Comment (bratanon) [inline custom_components/skodaconnect/const.py:32]: From #219 it is recommended to use 120 or 180. I guess 120 should be the minimum in this case?

Comment (dvx76) [reply custom_components/skodaconnect/const.py:32]: I don't see why the minimum should be raised. I've been running with 60 for a long time and have no issues. Also maybe someone wants to test something and have the poll frequency to 10s for a short period.

Raising the default to 120 makes sense given the recent reports from various users.

Comment (asd20752) [reply custom_components/skodaconnect/const.py:32]: Assuming Skoda has gone with 1000 request/day limit 70sec would be the closest to this rate limit and added 10 for good measure. 120 is a good value for default, but not taking away the possibility for the people who want to poll faster. 
The reason I suggest racing the minimum poll rate is to not give Skoda any reasons to limit the API more than necessary.

Comment (dvx76) [reply custom_components/skodaconnect/const.py:32]: Like I said, I have 60 and 0 problems with polling. There appear to be different rate limits depending on which endpoints are being hit, which depends on the car model. I'm sure a lot of people are running with 60 and I've heard some are running 30. They will be impacted by this change. I suggest 30 as the minimum value.

Comment (asd20752) [reply custom_components/skodaconnect/const.py:32]: Sounds good, will update the min value to 30 =)

**Pull request merged**